gst/nnstreamer/tensor_if/gsttensorif.c

```diff
@@ -370,6 +370,12 @@ gst_tensor_if_set_property_supplied_value (const GValue * value,
 {
   gint i;
   gboolean is_float = FALSE;
+
+  if (!value) {
```

Comment:
Whether value is null or not seems to be checked by g_value_get_string internal.
Do we need this?

Reply:
If we `use g_value_get_string()` like below, it generates error.
```c
  const gchar *param = g_value_get_string (NULL);
```
```bash
(process:23753): GLib-GObject-CRITICAL **: 13:39:03.002: g_value_get_string: assertion 'G_VALUE_HOLDS_STRING (value)' failed
```